### DISCUSSION TITLE
BROKEN - Contribution chart seems to be broken.

### DISCUSSION THREAD

**Discussion by hariacharya80** [community/community#59992] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I think I have found something unusual in the GitHub profile page. When you normally see the contribution chart, it will appear to be till the date contributions are made but as soon as you add some params to the url, something very unusual appears. It is the same case with all profiles. Please see this images
**Here is what seen normally :**
![Screenshot from 2023-07-04 04-56-05](https://github.com/community/community/assets/136120359/0aca3c4e-1323-481b-aeea-4639a75e3173)
**Here is what seen after some parms :**
![Screenshot from 2023-07-04 04-47-53](https://github.com/community/community/assets/136120359/c72aa019-2f52-45ba-b034-cfa610da0028)

Parms:
?tab=overview&from=2023-06-01&to=2023-06-30

Try:
https://github.com/HariAcharya80
https://github.com/HariAcharya80?tab=overview&from=2023-06-01&to=2023-06-30

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by chriskorosu** [comment, marked as answer]
> When you normally see the contribution chart, it will appear to be till the date contributions are made but as soon as you add some params to the url, something very unusual appears.

I'm assuming that by "unusual" you mean the commits that appear to have been made in the future. 

The chart is also laid out different, but that's probably because you specified one month of activity. I imagine the standard behavior is to generate the chart with the specified time-frame in the center.

> It is the same case with all profiles. 

Can't replicate on my own profile. The month is centered, but no additional commits after the present show up.

![image](https://github.com/community/community/assets/47898552/f88ff6f5-71f0-4236-b72c-1b557f9d7c0a)

I clicked some of the future commits that show up on your graph, and it seems they belong to a private repository. Do you have access to it to investigate further? 

![image](https://github.com/community/community/assets/47898552/7f4bd54a-5f3b-4d8e-8de1-23d21a142bf3)

It is possible to manually edit the date of a commit to make it appear as if it was contributed in the future. I suspect that's what's happening here, but can't be certain without access to the private repositories that are being referenced.

**Reply by hariacharya80** [reply]
![123](https://github.com/community/community/assets/136120359/10afe13b-7b50-428f-ad4c-0dcee58742b4)
I don't think so because I have no commits in the future and I'm confused how a newly created account would have future commits? I'm so confused.

Those private repos as per my knowledge have no commits in the future..
